site-icon-as-image/snippet.html

```diff
@@ -0,0 +1,18 @@
+<div id="siteIcon"></div>
+
+<script>
+  
+var siteLink = (s = $('.ui-dialog.focused').find('.sites_core').data('controllers')) &&
```

Comment:
Consider using the HTML widget as a starting point.
`var siteLink = (s = arguments[0].closest('.ui-dialog').find('.sites_core').data('controllers')) &&`